libmc/_client.pyx

```diff
@@ -282,11 +282,11 @@ cdef bytes _encode_value(object val, int comp_threshold, flags_t *flags):
 
 def encode_value(object val, int comp_threshold):
     cdef flags_t flags
-    cdef bytes = _encode_value(val, comp_threshold, &flags)
```

Comment:
That's depend on the return value of `_encode_value` (i.e.: the type of `bytes` is `bytes`). :sob: